dev-docs/source/Testing/SliceViewer/SliceViewer.rst

```diff
@@ -25,11 +25,12 @@ Data
 
 	from mantid.simpleapi import *
 
-	md_4D = CreateMDWorkspace(Dimensions=4, Extents=[-1,1,-1,1,-1,1,-10,10], Names="H,K,L,E", Units="U,U,U,V")
+	md_4D = CreateMDWorkspace(Dimensions=4, Extents=[-1,1,-1,1,-1,1,-10,10],
+        Names="H,K,L,E", Frames='HKL,HKL,HKL,General Frame',Units='r.l.u.,r.l.u.,r.l.u.,meV')
 	FakeMDEventData(InputWorkspace=md_4D, PeakParams='500000,0,0,0,0,3') # 4D data
 	# Create a histogrammed (binned) workspace with 100 bins in each of the H, K and L dimensions
-	histoWS_3D = BinMD(InputWorkspace=md_4D, AlignedDim0='H,-1,1,100', AlignedDim1='K,-1,1,100', AlignedDim2='L,-1,1,100') # 3D cut
-
+	mdHisto_3D = BinMD(InputWorkspace=md_4D, AlignedDim0='H,-1,1,100', AlignedDim1='K,-1,1,100', AlignedDim2='L,-1,1,100') # 3D cut
+    histoWS_2D = BinMD(InputWorkspace=md_4D, AlignedDim0='H,-1,1,100', AlignedDim1='K,-1,1,100') # 2D cut
```

Comment:
If we are taking a 2D cut here do we still need the instruction above about loading in ```CNCS_7860_event.nxs``` from the Training Course data? (line 20)

Reply:
It's probably still good to have both Matrix Workspaces: the CNCS data (which is event) and the SXD data (which is histo). Note the 2D cut is still an MD object which is different from the above (perhaps the confusion is due to a typo in the variable name, I intended it to be `mdHisto_2D`). I will make both points clearer.

Reply:
Thanks for making this clearer.